### ISSUE TITLE
Loading issue with 1.12.2.3167

### ISSUE THREAD

**Issue by nouuuugat** [Galileo88/DART-Project#6]
I've been unable to load the DART mod. I've validated KSP and there were some failed files which were reacquired, but this still hasn't solved the issue. The only relevant file I was able to find was KSP.log; there was no Logs/DART/ folder, and I couldn't locate ModuleManager.ConfigCache. I've uploaded a screenshot of the error it throws as well.
0
![KSPError](https://user-images.githubusercontent.com/93847113/140628517-aa51b1e2-0952-4190-9adf-d5270f9f221a.png)

 [KSP.log](https://github.com/Galileo88/DART-Project/files/7491484/KSP.log)

**Comment by R-T-B** [comment]
It seems unable to open the shader bundle.  It's possible your install is still corrupt, or the DART install itself needs to be replaced (particularly the shader files).

If that fails, it may just be your GPU can't run our asteroid shaders....  I doubt that though.  What is your GPU?

**Comment by nouuuugat** [comment]
I'm on a GeForce GTX 1050 on a Samsung Odyssey laptop

**Comment by R-T-B** [comment]
Sorry, not really sure what the issue is here.  I would try updating your gpu drivers and completely removing and then reinstalling DART, in that order.

Wish I could offer better advice.  Good luck.

**Comment by kspfan** [comment]
I'm having this exact same issue. My graphics drivers are completely up to date, and I have tried everything suggested above. I have also tried uninstalling and reinstalling DART several times and nothing has changed.

**Comment by R-T-B** [comment]
So odd.  Hmm.  Can I see your ksp.log too?  Maybe it'll reveal something the other one missed.

**Comment by kspfan** [comment]
Sure. Where is ksp.log located?

**Comment by R-T-B** [comment]
Should be in ksp install folder.

**Comment by kspfan** [comment]
Which folder is the "install folder?" I can't find anything with that name.

**Comment by R-T-B** [comment]
It is usually named "Kerbal Space Program."  What OS?  I can direct you to where it is usually found.

**Comment by kspfan** [comment]
I got the game off of Steam. I don't know if the folder exists, and if it does, I can't find it. I'm also not sure how to tell what OS I'm on.

**Comment by R-T-B** [comment]
Steam is pretty easy.  I believe if you right click a game in your library and select "properties" a dialog will come up that, amonst other things, has an option to open the install folder (they may call it the "data" folder).

**Comment by kspfan** [comment]
Okay, I think I found it
[Player.zip](https://github.com/Galileo88/DART-Project/files/7534306/Player.zip)
.

**Comment by R-T-B** [comment]
Yep, you did great!  That is not ksp.log, but player.log, which actually offers some more insight into what is happening early in the load process.  :)

I will look at this later today.

**Comment by kspfan** [comment]
I apologize for asking so late, but have you found anything wrong?

**Comment by R-T-B** [comment]
No need to apologize, I sort of dropped the ball on this, so thanks for the reminder!

I'll look again later today and update you.

**Comment by R-T-B** [comment]
According to the log, there is some body in settings.cfg that is not usually present.

If you've truly uninstalled all other potentially incompatible mods, you should be able to fix this by deleting settings.cfg and simply resetting your setttings manually.  Let me know if that works.
